PYME/LMVis/pipeline.py

```diff
@@ -674,7 +674,10 @@ def _ds_from_file(self, filename, **kwargs):
                 #old style matlab import
                 ds = tabular.MatfileSource(filename, kwargs['FieldNames'], kwargs['VarName'])
             else:
-                ds = tabular.MatfileColumnSource(filename)
```

Comment:
Much of the new logic in the modified `MatfileColumnSource` could be replaced with the following.

```python
ds = tabular.MatfileColumnSource(filename)

field_names = kwargs.get('FieldNames', None)
if field_names:
    ds = tabular.MappingFilter(ds, **{new_field : old_field for new_field, old_field in zip(field_names, ds.keys())})
```

Reply:
Yep. Changed.